### PR TITLE
C1 Qbo dashboard content fix

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

**Comment by DhaaraniCIT** [comment]
Screenshot 2024-04-04 at 2 18 21 PM

File: src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h5 *ngIf="groupedErrorStat.CATEGORY_MAPPING" class="tw-pt-4-px tw-text-text-mut
                                 <h5 class="!tw-text-text-muted tw-text-14-px tw-pt-10-px !tw-font-400 !tw-leading-4"><p>Map category in {{brandingConfig.brandName}} to their corresponding records in {{appName}}.</p>
                                 </h5>
                                 <div *ngIf="!groupedErrorStat.CATEGORY_MAPPING || (groupedErrorStat.CATEGORY_MAPPING && groupedErrorStat.CATEGORY_MAPPING.resolvedCount !== groupedErrorStat.CATEGORY_MAPPING.totalCount)">
-                                    <button type="button" (click)="showMappingResolve(ErrorType.CATEGORY_MAPPING, errors.CATEGORY_MAPPING, ExportErrorSourceType.CATEGORY)" class="p-button secondary-sm">
+                                    <button type="button" (click)="showMappingResolve(ErrorType.CATEGORY_MAPPING, errors.CATEGORY_MAPPING, ExportErrorSourceType.CATEGORY)" class="export-btn tw-float-right tw-flex tw-justify-end tw-items-center tw-text-white tw-text-500 tw-bg-mandatory-field-color tw-rounded-4-px p-button">
```
Comment (ashwin1111) [inline src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html:51]: same

File: src/app/shared/components/dashboard/dashboard-export-section/dashboard-export-section.component.html [hunk 2]
```diff
@@ -20,10 +20,7 @@ <h5 class="tw-text-14-px tw-font-400 tw-text-text-secondary tw-pt-8-px" [ngClass
                     </h5>
                     <h5 class="tw-text-14-px tw-font-400 tw-text-text-secondary tw-pt-8-px" [ngClass]="{'tw-pl-0': !brandingFeatureConfig.illustrationsAllowed, 'tw-pl-34-px': brandingFeatureConfig.illustrationsAllowed}" *ngIf="isExportInProgress">This may take a few minutes...</h5>
                     <h5 class="tw-text-14-px tw-font-400 tw-text-text-secondary tw-pt-8-px" [ngClass]="{'tw-pl-0': !brandingFeatureConfig.illustrationsAllowed, 'tw-pl-34-px': brandingFeatureConfig.illustrationsAllowed}" *ngIf="!isExportInProgress && !exportableAccountingExportIds.length && accountingExportSummary?.successful_accounting_export_count">
-                        No new expenses to export right now.
-                        <p class="tw-pt-4-px">
-                            Once expenses are {{ importStates }}, they'll automatically be available here for export.
-                        </p>
+                        There are no expenses to export right now. Once expenses are closed, they'll be available here for export.
```
Comment (ashwin1111) [inline src/app/shared/components/dashboard/dashboard-export-section/dashboard-export-section.component.html:23]: create a slack thread for this, cc Shwetabh as well. We cannot use "closed" keyword here - think about customer setting up import state as "Approved"

File: src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html [hunk 3]
```diff
@@ -80,7 +80,7 @@ <h4 class="tw-text-14-px !tw-font-500 tw-text-text-secondary tw-w-600-px">{{ err
                                 <h5 class="!tw-text-text-muted tw-text-12-px tw-pt-10-px !tw-font-400 !tw-leading-4 tw-w-600-px"><p>{{ error.error_detail }}</p>
                                 </h5>
                                 <div>
-                                    <button type="button" (click)="showErrorDialog(error)" class="p-button outline-sm">
+                                    <button type="button" (click)="showErrorDialog(error)" class="export-btn tw-float-right tw-flex tw-justify-end tw-items-center tw-text-white tw-text-500 tw-bg-mandatory-field-color tw-rounded-4-px p-button">
```
[the remark this answers is not in the document]
Comment (ashwin1111) [reply src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html:83]: @DhaaraniCIT we can use `primary-outline` here, it should look same as Refresh data button. Can copy whatever style is used there

File: src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html (hunk 1 above)
Comment (ashwin1111) [reply src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html:51]: looks good

File: src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html [hunk 4]
```diff
@@ -18,13 +18,13 @@ <h5 class="!tw-text-text-muted tw-text-14-px !tw-font-400 !tw-leading-4 tw-pt-10
...
                             <div class="flex-wrapper">
                                 <h5 class="!tw-text-text-muted tw-text-14-px tw-pt-10-px !tw-font-400 !tw-leading-4">
                                     <p>Map employees in {{brandingConfig.brandName}} to their corresponding records in {{appName}}.</p>
                                 </h5>
                                 <div *ngIf="!groupedErrorStat.EMPLOYEE_MAPPING || (groupedErrorStat.EMPLOYEE_MAPPING && groupedErrorStat.EMPLOYEE_MAPPING.resolvedCount !== groupedErrorStat.EMPLOYEE_MAPPING.totalCount)">
-                                    <button type="button" (click)="showMappingResolve(ErrorType.EMPLOYEE_MAPPING, errors.EMPLOYEE_MAPPING, ExportErrorSourceType.EMPLOYEE)" class="p-button secondary-sm">
+                                    <button type="button" (click)="showMappingResolve(ErrorType.EMPLOYEE_MAPPING, errors.EMPLOYEE_MAPPING, ExportErrorSourceType.EMPLOYEE)" class="export-btn tw-float-right tw-flex tw-justify-end tw-items-center tw-text-white tw-text-500 tw-bg-mandatory-field-color tw-rounded-4-px p-button">
```
[the remark this answers is not in the document]
Comment (ashwin1111) [reply src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html:27]: looks good

File: src/app/shared/components/dashboard/dashboard-export-section/dashboard-export-section.component.html [hunk 5]
```diff
@@ -20,7 +20,7 @@ <h5 class="tw-text-14-px tw-font-400 tw-text-text-secondary tw-pt-8-px" [ngClass
                     </h5>
                     <h5 class="tw-text-14-px tw-font-400 tw-text-text-secondary tw-pt-8-px" [ngClass]="{'tw-pl-0': !brandingFeatureConfig.illustrationsAllowed, 'tw-pl-34-px': brandingFeatureConfig.illustrationsAllowed}" *ngIf="isExportInProgress">This may take a few minutes...</h5>
                     <h5 class="tw-text-14-px tw-font-400 tw-text-text-secondary tw-pt-8-px" [ngClass]="{'tw-pl-0': !brandingFeatureConfig.illustrationsAllowed, 'tw-pl-34-px': brandingFeatureConfig.illustrationsAllowed}" *ngIf="!isExportInProgress && !exportableAccountingExportIds.length && accountingExportSummary?.successful_accounting_export_count">
-                        There are no expenses to export right now. Once expenses are closed, they'll be available here for export.
+                        There are no expenses to export right now. Once expenses are {{ importStates }}, they'll be available here for export.
```
Comment (ashwin1111) [inline src/app/shared/components/dashboard/dashboard-export-section/dashboard-export-section.component.html:23]: fix same content in line 16 and 23

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**